src/Libraries/RevitNodesUI/Controls/ComboControl.xaml

```diff
@@ -31,7 +31,9 @@
         </Button>
         <ComboBox Grid.Row="1"
                   ItemsSource="{Binding Model.DropDownNodeModel.Items}"
-                  SelectedIndex="{Binding Model.SelectedIndex, Mode=TwoWay}"/>
+                  SelectedIndex="{Binding Model.SelectedIndex, Mode=TwoWay}"
+                  Style="{StaticResource RefreshComboBox}"
+                  Margin="0,0,0,0"/>
```

Comment:
for all 0s, do we need this line?

Reply:
Yes, we need this to reset the Margin value which set by RefreshComboBox here https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml#L1356

Reply:
I see, thanks for the explanation!